yapapi/runner/__init__.py

```diff
@@ -207,6 +212,9 @@ async def map(
     ) -> AsyncIterator[Task[D, R]]:
         """Run computations on providers.
 
+        In other words, map the computation's inputs specified by task data
+        to the ouput resulting from running the execution script on a providers' execution units.
```

Comment:
`ouput` -> `output`
I would rename this to `run_on_providers` or maybe to `run`. Sorry, `map` is a legacy name which has to be changed. Instead of renaming it, we decided to make this function work similar to the `map` function used in the functional programming paradigm – but we do not preserve the order of tasks.

Reply:
sure, we can do that, but not in this version here

Reply:
what I wanted to do here is just to add some docs to explain the name just so it doesn't sound so much out of place

Reply:
I know. It's OK.

Reply:
@shadeofblue Nice try!